dev-docs/source/EventWorkspaceDev.rst

```diff
@@ -0,0 +1,93 @@
+.. _Event Workspace Dev:
+
+===========================
+Event Workspace Development
+===========================
+
+.. contents::
+  :local:
+
+The following information will be useful to you if you want to write an
+:ref:`algorithm` that is :ref:`Event Workspace <EventWorkspace>` aware.
+
+Individual Neutron Event Data (TofEvent)
```

Comment:
It may be useful to mention what header ([Framework/Types/inc/MantidTypes/Event/TofEvent.h](https://github.com/mantidproject/mantid/blob/master/Framework/Types/inc/MantidTypes/Event/TofEvent.h)) has these constructs so people can find more information. There are actually a couple of different types of "events" including `TofEvent`, `WeightedEvent`, and `WeightedEventNoTime` which all have different number of fields. They are all implemented as `struct` rather than classes.